### ISSUE TITLE
A released product

### ISSUE THREAD

**Issue by MGParisi** [tayfunerbilen/jquery-to-javascript-convert#9]
I worked on a script like this.  Didn't see your script.  Very cool.  Try it out, please as I'm hosting it for free.
https://properprogramming.com/jquery-to-javascript-converter/

**Comment by dipanshumahla** [comment]
It returns exactly the same output as input.

**Comment by MGParisi** [comment]
Yep, it's fixed.  Sorry for that.  No one even complained.  A missing reference on a return value was causing it.

**Comment by MGParisi** [comment]
I was thinking of hosting this solution on my website as well.

**Comment by dipanshumahla** [comment]
A lot of people search for jquery to javascript converter. That would be a good idea. Let us know the link to your website.
